get.go

```diff
@@ -70,7 +71,10 @@ var DefaultClient = &Client{
 
 func init() {
 	httpGetter := &HttpGetter{
-		Netrc: true,
+		Netrc:                 true,
+		XTerraformGetDisabled: true,
+		HeadFirstTimeout:      10 * time.Second,
```

Comment:
🤔 Maybe we should go with `DoNotCheckHeadFirst`, to avoid the initial `HEAD` request altogether.

```suggestion
		DoNotCheckHeadFirst:   true,
```

What do you think?

Reply:
Packer uses the default Getter settings with a few additional Getters. I updated them to match what we plan to use in Packer.  That said, I think Packer could take advantage of the Range header. So I opted for the HeadFirstTimeout over the disabling it entirely. Does that sound reasonable to you?

Reply:
Given the need of the changes. I went ahead and merged as is. If after working through Packer further we find we can disable entirely. We can revisit. But sticking with this default for now.